src/applications/edu-benefits/tests/components/SchoolSelectField.unit.spec.jsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import {
+  shallow,
+  mount
+} from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import { SchoolSelectField } from '../../components/SchoolSelectField';
+
+describe('<SchoolSelectField>', () => {
+  it('should render initial search view', () => {
+    const tree = shallow(<SchoolSelectField
+      formContext={{}}
+      showInstitutions={false}
+      showInstitutionsLoading={false}
+      showPagination={false}
+      showPaginationLoading={false}/>
+    );
+
+    expect(tree.find('.search-controls').exists()).to.be.true;
+    expect(tree.find('#root_school_view:manualSchoolEntry_name').exists()).to.be.false;
+    expect(tree.find('.institution-name').exists()).to.be.false;
+    expect(tree.find('.loading-indicator-container').exists()).to.be.false;
+    expect(tree.find('.va-pagination').exists()).to.be.false;
+  });
+
+  it('should render institution results view', () => {
+    const institutions = [{
+      city: 'testcity',
+      facilityCode: 'test',
+      name: 'testName',
+      state: 'testState'
+    }];
+    const tree = mount(<SchoolSelectField
+      formContext={{}}
+      currentPageNumber={1}
+      institutionQuery="test"
+      institutions={institutions}
+      pagesCount={2}
+      searchInputValue="test"
+      searchResultsCount={1}
+      showInstitutions
+      showInstitutionsLoading={false}
+      showPagination
+      showPaginationLoading={false}/>
+    );
+
+    expect(tree.find('.search-controls').exists()).to.be.true;
+    expect(tree.find('.institution-name').exists()).to.be.true;
+    expect(tree.find('.institution-city-state').exists()).to.be.true;
+    expect(tree.find('.va-pagination').exists()).to.be.true;
+    expect(tree.find('.loading-indicator-container').exists()).to.be.false;
+    expect(tree.find('#root_school_view:manualSchoolEntry_name').exists()).to.be.false;
```

Comment:
Should we add a test for when the manual entry should be visible?

Reply:
I figured the definition tests cover that.